iothub/service/src/PurgeMessageQueueResult.cs

```diff
@@ -2,10 +2,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // ---------------------------------------------------------------
 
+using Newtonsoft.Json;
```

Comment:
why did you need to add this using?

Reply:
I didn't. Moved it from inside the namespace to outside